dropwizard-db/src/test/java/io/dropwizard/db/DataSourceConfigurationTest.java

```diff
@@ -107,6 +109,17 @@ public void testInlineUserPasswordConfiguration() throws Exception {
         assertThat(ds.getUser()).isNull();
         assertThat(ds.getPassword()).isNull();
     }
+    // Test added for https://github.com/dropwizard/dropwizard/issues/1037
+    @Test
+    public void testInitialSizeZeroIsAllowed() throws Exception {
+        try {
+            DataSourceFactory ds = getDataSourceFactory("yaml/empty_initial_pool.yml");
```

Comment:
In general I'd agree, but that argument is kind of moot in a test consisting of a single statement (and one statement for assertions).